src/api/app/models/unregistered_user.rb

```diff
@@ -44,7 +44,7 @@ def self.can_register?
   def self.register(opts)
     can_register?
 
-    opts[:note] = nil unless User.current && User.current.is_admin?
+    opts[:note] = nil unless User.current && User.admin_session?
```

Comment:
With User#admin_session? defined as `current && current.is_admin?` this doesn't make sense:wink:

Reply:
hmm, not sure how I missed it